lib/Twig/Error/Loader.php

```diff
@@ -24,8 +24,18 @@
  */
 class Twig_Error_Loader extends Twig_Error
 {
-    public function __construct($message, $lineno = -1, $filename = null, Exception $previous = null)
+    private $guess = false;
+
+    public function __construct($message, $lineno = -1, $source = null, Exception $previous = null)
+    {
+        parent::__construct($message, false, null, $previous);
+        $this->guess = true;
```

Comment:
I'm confused by this. We should never guess here, but you set it to `true` and I don't anywhere in the code a way to turn it off. Can you explain?